### ISSUE TITLE
Inconsistent use of short-hand config option

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#320]
Almost all benchmark command have `--config` for an option, but the short-hand version varies between commands.  For example, for `pbench-user-benchmark` the short-hand is `-C`, but for `pbench-uperf` it is `-c`.

I think we should make this behavior consistent by dropping the short-hand across the board, but doing by accepting the short-hand for a few releases, emitting a warning that the caller should use `--config` instead.

**Comment by ndokos** [comment]
pbench-fio uses -j :-) probably cut-n-paste error

**Comment by atheurer** [comment]
I agree, let's get rid of all shorthand.  Some will revolt, but I don't see a better solution.
